app/src/main/java/tttTDD/Java/GameInput.java

```diff
@@ -0,0 +1,50 @@
+package tttTDD.Java;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class GameInput {
+
+    public static int playerSelection(Scanner input, Display display) {
+        int value;
+
+        try {
+            value = input.nextInt();
+            if (value > 0 || value < 4) {
+                return value;
+            }
+        }
+
+        catch (Exception e) {
+            System.out.println(display.wrongTypeMessage());
```

Comment:
How are you deciding between calling `System.out.println()` directly versus `display.render()`